NetTopologySuite.IO.GeoTools/ShapefileDataWriter.cs

```diff
@@ -22,11 +22,11 @@ public class ShapefileDataWriter
         /// <param name="feature">The feature.</param>
         /// <param name="count">The count.</param>
         /// <returns></returns>
-        public static DbaseFileHeader GetHeader(IFeature feature, int count)
+        public static DbaseFileHeader GetHeader(IFeature feature, int count, Encoding encoding = null)
```

Comment:
> but that breaks the API...

Adding a default parameter is also a breaking change, at least in terms of binary compatibility.

Overload instead?